en/docs/learn/consume-api/account-management-dev-portal/recover-dev-portal-password.md

```diff
@@ -0,0 +1,18 @@
+# Recover Password for the Developer Portal
+
+!!! note
+    The password recovery feature does not work out-of-the-box because an email server needs to be configured to be able to send the password recovery email. For more information, see Enable Password Recovery for the Developer Portal.
```

Comment:
see Enable Password Recovery for the Developer Portal - > we need to link to the topic

Reply:
fixed